config/cohere.yaml

```diff
@@ -0,0 +1,75 @@
+# ==================================================================
+#            Configuration file for Canopy Server with Cohere.
+# ==================================================================
+
+# ---------------------------------------------------------------------------------
+system_prompt: &system_prompt |
+  Use the documents to answer the user question at the next messages. The documents are retrieved from a knowledge
+  database and you should use only the facts from the documents to answer. Always remember to include the source to
+  the documents you used from their 'source' field in the format 'Source: $SOURCE_HERE'.
+  If you don't know the answer, just say that you don't know, don't try to make up an answer, use the documents.
+  Don't address the documents directly, but use them to answer the user question like it's your own knowledge.
+
+
+# -------------------------------------------------------------------------------------------
+# Tokenizer configuration
+# -------------------------------------------------------------------------------------------
+tokenizer:
+  type: CohereHFTokenizer
+  params:
+    model_name: Cohere/Command-nightly
+
+
+# -------------------------------------------------------------------------------------------------------------
+# Chat engine configuration
+# -------------------------------------------------------------------------------------------------------------
+chat_engine:
+  params:
+    system_prompt: *system_prompt
+
+  # -------------------------------------------------------------------------------------------------------------
+  # LLM configuration
+  # -------------------------------------------------------------------------------------------------------------
+  llm: &llm
```

Comment:
Can we make sure that the value for max tokens make sense here for Cohere API? it also seems like it make sense to have the default a huge number, and use `prompt_truncation=True` to let Cohere manage the truncation

Reply:
Using `max_tokens=None` the model will generate a completion until EOS. Aside from that, no need for a default number to be suggested; up to user's discretion imo. `prompt_truncation = AUTO` is also the API default so would happen automatically.